altair/examples/stripplot.py

```diff
@@ -0,0 +1,39 @@
+"""
+Stripplot
+-----------------
+This example shows how to make a Stripplot.
+"""
+# category: scatter plots
+import altair as alt
+from vega_datasets import data
+
+source = data.movies.url
+
+stripplot =  alt.Chart(source, width=40).mark_circle(size=8).encode(
+    x=alt.X(
+        'jitter:Q',
+        title=None,
+        axis=alt.Axis(values=[0], ticks=True, grid=False, labels=False),
+        scale=alt.Scale(),
+    ),
+    y=alt.Y('IMDB_Rating:Q'),
+    color=alt.Color('Major_Genre:N', legend=None),
+    column=alt.Column(
+        'Major_Genre:N',
+        header=alt.Header(
+            labelAngle=-90,
+            titleOrient='top',
+            labelOrient='bottom',
+            labelAlign='right',
+            labelPadding=3,
+        ),
+    ),
+).transform_calculate(
+    jitter='pow(random()-0.5,3)'
```

Comment:
This looks a bit better IMO with gaussian noise, which you can generate with a Box-Muller transform:
```python
transform_calculate(
   # Generate Gaussian jitter with a Box-Muller transform
   jitter='sqrt(-2*log(random()))*cos(2*PI*random())'
)
```